WcaOnRails/app/models/team_member.rb

```diff
@@ -51,6 +55,34 @@ def cannot_ban_user_with_upcoming_comps
     end
   end
 
+  def status
+    if leader?
+      TEAM_STATUS_LEADER
+    elsif senior_member?
+      TEAM_STATUS_SENIOR_MEMBER
+    else
+      TEAM_STATUS_MEMBER
+    end
+  end
+
+  def role
+    {
+      start_date: start_date,
+      is_active: current_member?,
+      group: {
+        id: "teams_committees_" + team.id.to_s,
```

Comment:
Isn't this prefix a bit redundant with the `group_type: UserGroup.group_types[:teams_committees]` below?